channels/channel.be/vier/chn_vier.py

```diff
@@ -587,7 +587,7 @@ def __extract_stream_collection(self, item, video_info):
         prefer_widevine = True  # video_info.get("videoType") != "longForm"
 
         if stream_collection:
-            drm_key = stream_collection["drmKey"]
+            drm_key = stream_collection["drmProtected"]
```

Comment:
I always prefer to not break any backward compatibility. Some broadcasters keep old streams somewhere with older content. So please use `drm_key = stream_collection.get("drmProtected")` and add `has_drm = stream_collection.get("drmProtected", False)`